docs/architecture/cloud-native/combine-containers-serverless-approaches.md

```diff
@@ -28,9 +28,17 @@ func init ProjectName --worker-runtime dotnet --docker
 
 When the project is created, it will include a Dockerfile and the worker runtime configured to `dotnet`. Now, you can create and test your function locally. Build and run it using the  `docker build` and `docker run` commands. For detailed steps to get started building Azure Functions with Docker support, see the [Create a function on Linux using a custom image](https://docs.microsoft.com/azure/azure-functions/functions-create-function-linux-custom-image) tutorial.
 
+When creating Azure functions, you'll want to reference version 3 of the Azure Functions runtime, which targets .NET Core 3.1 and later. [This link](https://docs.microsoft.com/azure/azure-functions/functions-versions) describes the runtime versions available for Azure Functions.
```

Comment:
Remove duplicate content.

Reply:
So, think you suggesting removing the item in green. It's definitely misplaced. This paragraph should not be under the heading "Combine serverless and docker containers." Would it make sense to move this somewhere else in the chapter, as it talks runtime versions for Functions?  Let me know what you think.

Reply:
Never mind - removed the paragraph.